index.html

```diff
@@ -8,6 +8,62 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    Testing CSS is linked Testing pull worked
+    <!-- Header Section  -->
+
+    <header>
+      <div class="header-grid">
+        <div class="Twitter-logo">
+          <img src="" alt="" />
+        </div>
+
+        <div class="home-grid">
+          <div class="home"></div>
+          <div class="stars-icon"></div>
+        </div>
+
+        <div class="search"></div>
+      </div>
+    </header>
+
+    <!-- Main Section  -->
+
+    <main>
+      <div class="main-section-grid">
+        <div class="left-menu">
+          <ul>
+            <div class="home"></div>
+            <div class="explore"></div>
+            <div class="notifications"></div>
+            <div class="messages"></div>
+            <div class="bookmarks"></div>
+            <div class="lists"></div>
+            <div class="profile"></div>
+            <div class="more"></div>
+          </ul>
+
+          <div class="tweet"></div>
+
+          <div class="profile-info"></div>
+        </div>
+
+        <div class="middle-main">
+          <div class="make-tweet">
+            <div class="type-tweet"></div>
+            <div class="tweet-icons"></div>
+            <div class="tweet-button"></div>
+          </div>
+
+          <div class="main-feed"></div>
+        </div>
+      </div>
+
+      <div class="right-column">
+        <div class="whats-happening"></div>
+      </div>
+    </main>
+
+    <!-- Footer Section - Not needed -->
+
+    <!-- <Footer></Footer> -->
```

Comment:
Footer not needed, therefore not needed to include. Could clutter code.  Deleted at the end, but probably not needed at all.